### DISCUSSION TITLE
How to set a group avatar background to transparency?

### DISCUSSION THREAD

**Discussion by zengjunwen** [libvips/libvips#2809] [Q&A]
![9e3f5f1c88b85896f5f558f24e29c6930b030100000061bd0000000002010079_80](https://user-images.githubusercontent.com/21982782/168811648-91b9c84d-7973-463b-9627-5dc5e373fb7c.jpeg)
I use arrayjoin to generate a group avatar. e.g.I use arrayjoin to generate a group avatar. How can I set the background to transparency? thx

C lang:
`int ret = vips_arrayjoin(in, out, n, "across", across, "halign",1, "shim", shim, "background", vipsBackground,NULL);`

**Comment by jcupitt** [comment]
Hi, just save as PNG (probably).

Post some sample images and test code if you need more help.

**Reply by zengjunwen** [reply]
thx. saving as PNG is ok!
